### PR TITLE
KOGITO-2104: Configure storybook on common package

### PR DESCRIPTION
**Pull request by Sara4994**
Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [ ] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Comment by cristianonicolai** [comment]
jenkins please retest this

File: ui-packages/packages/common/package.json [hunk 1]
```diff
@@ -16,12 +16,29 @@
...
     "test:coverage": "yarn test --coverage",
+    "storybook": "start-storybook -p 9001 -c .storybook",
     "lint": "tslint -c ./tslint.json --project . './src/**/*.ts{,x}'",
     "format": "prettier --config ../../.prettierrc --check --write './src/**/*.{tsx,ts,js}'",
     "clean": "rimraf dist",
     "codegen": "graphql-codegen"
   },
   "dependencies": {},
+  "devDependencies": {
+    "@storybook/addon-a11y": "^5.3.19",
```
Comment (cristianonicolai) [inline ui-packages/packages/common/package.json]: @Sara4994 please use fixed versions.

File: ui-packages/packages/common/src/graphql/queries.tsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import gql from 'graphql-tag';
-
+// @ts-ignore
```
Comment (cristianonicolai) [inline ui-packages/packages/common/src/graphql/queries.tsx]: @Sara4994 shouldnt this file be excluded in tsconfig?

File: ui-packages/packages/common/src/static/managementConsoleLogo.svg [hunk 3]
```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (cristianonicolai) [inline ui-packages/packages/common/src/static/managementConsoleLogo.svg]: @Sara4994 I think these logos should remain in their own repos. Each place defines its own svg when using the brand component.

File: ui-packages/packages/common/src/graphql/queries.tsx (hunk 2 above)
Comment (Sara4994) [reply ui-packages/packages/common/src/graphql/queries.tsx]: done

File: ui-packages/packages/common/package.json (hunk 1 above)
Comment (Sara4994) [reply ui-packages/packages/common/package.json]: done

File: ui-packages/packages/common/src/static/managementConsoleLogo.svg (hunk 3 above)
Comment (Sara4994) [reply ui-packages/packages/common/src/static/managementConsoleLogo.svg]: done

**Review by cristianonicolai — APPROVED** [review]

File: ui-packages/packages/common/.storybook/webpack.config.js [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+const STORIES_PATH = path.join(__dirname, '../stories');
+const webpack = require('webpack');
+
+module.exports = ({config}) => {
+  config.plugins.push(
+    new webpack.EnvironmentPlugin({
+      KOGITO_AUTH_ENABLED: false,
+      KOGITO_KEYCLOAK_REALM: 'kogito',
+      KOGITO_KEYCLOAK_URL: 'http://localhost:8280',
+      KOGITO_KEYCLOAK_CLIENT_ID: 'kogito-management-console',
```
Comment (pefernan) [inline ui-packages/packages/common/.storybook/webpack.config.js]: Just nitpicking... is right to have a reference to management-console in common?

**Review by pefernan — APPROVED** [review]
+1

**Review by AjayJagan — APPROVED** [review]
:+1:

File: ui-packages/packages/common/.storybook/webpack.config.js (hunk 4 above)
Comment (cristianonicolai) [reply ui-packages/packages/common/.storybook/webpack.config.js]: @Sara4994 are these property actually used?

Comment (Sara4994) [reply ui-packages/packages/common/.storybook/webpack.config.js]: @cristianonicolai  below env variables are use on AboutModalBox.tsx which will be displayed on storybook as well, have removed other unused variables.

KOGITO_DATAINDEX_HTTP_URL: 'http://localhost:4000/graphql',
      KOGITO_APP_VERSION: 'DEV',
      KOGITO_APP_NAME: 'Kogito Console'

File: ui-packages/packages/common/src/static/managementConsoleLogo.svg (hunk 3 above)
Comment (jstastny-cz) [reply ui-packages/packages/common/src/static/managementConsoleLogo.svg]: I can still see the svg in common package ... what was the conclusion here?

Comment (cristianonicolai) [reply ui-packages/packages/common/src/static/managementConsoleLogo.svg]: @jstastny-cz management console svg was added in a _examples_ folder so at least the storybook has some content to show in the component but we avoid adding this file into the final build/dist

**Review by jstastny-cz — APPROVED** [review]

**Review by jstastny-cz — CHANGES_REQUESTED** [review]
Ah, one more thing - the coverage report is failing - it's considering stories.tsx files and should not .... we should add exclusions to jest and sonar.

**Comment by Sara4994** [comment]
> Ah, one more thing - the coverage report is failing - it's considering stories.tsx files and should not .... we should add exclusions to jest and sonar.

 i see jest is not considering stories.tsx files and i dont see stories.tsx files on the coverage report.. May be something to do with Sonar?

**Review by jstastny-cz — APPROVED** [review]

**Review by nmirasch — APPROVED** [review]

**Comment by pefernan** [comment]
Thanks @Sara4994

**Pull request merged**